tests/php/sync/test_class.jetpack-sync-full.php

```diff
@@ -388,13 +388,16 @@ function test_full_sync_sends_all_post_meta() {
 
 		$this->assertEquals( 'foo', $this->server_replica_storage->get_metadata( 'post', $post_id, 'test_meta_key', true ) );
 		$this->assertEquals( array( 'foo', 'bar' ), $this->server_replica_storage->get_metadata( 'post', $post_id, 'test_meta_array', true ) );
+
+		Jetpack_Sync_Settings::update_settings( array( 'post_meta_whitelist' => array( ) ) );
```

Comment:
Is this line necessary for the tests to pass? If so we have an issue with how we're resetting data.